### PR TITLE
send nick change from irc to discord

### PR DESCRIPTION
**Pull request by Junky2k**
anoy discord user with the nick changes of the irc users

### CODE REVIEW

File: lib/bot.js [hunk 1]
```diff
@@ -125,6 +125,20 @@ class Bot {
       this.sendToDiscord(author, to, `*${text}*`);
     });
 
+    this.ircClient.on('nick', (oldnick, newnick, channels) => {
```
Comment (ekmartin) [inline lib/bot.js]: `oldNick` and `newNick` maybe? Nitpick :)

**Comment by Throne3d** [comment]
The only thing I would be wary about is if the library sends the list of channels containing *every* channel, like it does for quit, in which case this (I think) will send to all channels, even ones the user isn't in.

I haven't actually tried it out in practice yet so this might be a nonissue.

**Comment by Throne3d** [comment]
This should work fine, I think.

**Comment by ekmartin** [comment]
Thanks!

**Pull request merged**